src/Sulu/Bundle/ContactBundle/Resources/translations/admin.de.json

```diff
@@ -41,5 +41,7 @@
     "sulu_contact.upload_avatar_info_text": "Bild ablegen oder klicken um den Avatar zu ändern",
     "sulu_contact.upload_logo_info_text": "Bild ablegen oder klicken um das Logo zu ändern",
     "sulu_contact.no_contact_selected": "Kein Kontakt ausgewählt",
-    "sulu_contact.single_selection_overlay_title": "Kontakt auswählen"
+    "sulu_contact.single_contact_selection_overlay_title": "Kontakt auswählen",
+    "sulu_contact.no_accounnt_selected": "Keine Organisation ausgewählt",
```

Comment:
`no_account_selected` is spelled wrong, and therefore does not translate correctly in the UI.